forge-game/src/main/java/forge/game/card/Card.java

```diff
@@ -2390,10 +2394,14 @@ else if (o.grantsZonePermissions())
             sb.append("(Gain the next level as a sorcery to add its ability.)").append(linebreak);
         }
 
-        if (type.hasSubtype("Saga")) {
-            sb.append("(As this Saga enters and after your draw step, add a lore counter.");
+        // FIXME: Use state.hasKeyword() ? But need to add Read ahead in keyword enum ?
+        // Check if the saga card has the keyword Read ahead
+        Collection<KeywordInterface> keywords = getUnhiddenKeywords();
+        boolean readAhead = keywords.toString().contains("Read ahead");
+        if (type.hasSubtype("Saga") && !readAhead) {
+            sb.append("(").append(Localizer.getInstance().getMessage("lblSagaHeader"));
```

Comment:
I don't really know if this is the right way to check if the saga card has read ahead keyword.
I assume I need to use hasKeyword() method, but for that need to add the read ahead keyword in the enum class Keyword. But don't really know if this is suppose to be there ?

Reply:
Try using `hasStartOfKeyword`